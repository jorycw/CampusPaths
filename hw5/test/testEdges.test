# Testing file.
# Tests adding multiple edges from a single node, 
# including one to itself, then prints edges out. 

# Create graphs
CreateGraph graph1
CreateGraph graph2

# Adds 3 different nodes to graph1
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Adds 3 different nodes to graph2
AddNode graph2 m1
AddNode graph2 m2
AddNode graph2 m3

# Adds 4 edges to graph1
AddEdge graph1 n1 n1 toMyself
AddEdge graph1 n1 n3 n1TOn3
AddEdge graph1 n1 n3 xtran1TOn3
AddEdge graph1 n1 n2 whyNotAddN2

# Adds 3 edges to graph 2
AddEdge graph2 m1 m3 sameNames
AddEdge graph2 m1 m2 sameNames
AddEdge graph2 m2 m1 notAChildofN1

# Prints Edges of n1 in graph1 and m1 in graph2
ListChildren graph1 n1
ListChildren graph2 m1
