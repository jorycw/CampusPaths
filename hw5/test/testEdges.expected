# Testing file.
# Tests adding multiple edges from a single node, 
# including one to itself, then prints edges out. 

# Create graphs
created graph graph1
created graph graph2

# Adds 3 different nodes to graph1
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Adds 3 different nodes to graph2
added node m1 to graph2
added node m2 to graph2
added node m3 to graph2

# Adds 4 edges to graph1
added edge toMyself from n1 to n1 in graph1
added edge n1TOn3 from n1 to n3 in graph1
added edge xtran1TOn3 from n1 to n3 in graph1
added edge whyNotAddN2 from n1 to n2 in graph1

# Adds 3 edges to graph 2
added edge sameNames from m1 to m3 in graph2
added edge sameNames from m1 to m2 in graph2
added edge notAChildofN1 from m2 to m1 in graph2

# Prints Edges of n1 in graph1 and m1 in graph2
the children of n1 in graph1 are: n1(toMyself) n2(whyNotAddN2) n3(n1TOn3) n3(xtran1TOn3)
the children of m1 in graph2 are: m2(sameNames) m3(sameNames)